### PR TITLE
info: Fix showing offline nodes in the pretty output

### PR DESCRIPTION
**Pull request by vadmeste**
mc admin info was not properly showing the offline status of an offline node.

This commit fixes it.

### CODE REVIEW

File: cmd/admin-info.go [hunk 1]
```diff
@@ -61,50 +61,65 @@ type clusterStruct struct {
...
 	// Dot represents server status, online (green) or offline (red)
 	dot := "●"
 	// Color palette initialization
 	console.SetColor("Info", color.New(color.FgGreen, color.Bold))
 	console.SetColor("InfoFail", color.New(color.FgRed, color.Bold))
+
 	// MinIO server type default
 	backendType := "Unknown"
+	// Set the type of MinIO server ("FS", "Erasure", "Unknown")
+	v := reflect.ValueOf(u.Info.Backend)
```
Comment (harshavardhana) [inline cmd/admin-info.go]: Backend is a proper type not sure why we need a type inference.. @vadmeste

Comment (vadmeste) [reply cmd/admin-info.go]: > Backend is a proper type not sure why we need a type inference.. @vadmeste

Recent changes removed backend type field in admin info strucutre and allocated a new backend data with interface type

```
// InfoMessage container to hold server admin related information.
type InfoMessage struct {
        Mode         string             `json:"mode,omitempty"`
        Domain       []string           `json:"domain,omitempty"`
        Region       string             `json:"region,omitempty"`
        SQSARN       []string           `json:"sqsARN,omitempty"`
        DeploymentID string             `json:"deploymentID,omitempty"`
        Buckets      Buckets            `json:"buckets,omitempty"`
        Objects      Objects            `json:"objects,omitempty"`
        Usage        Usage              `json:"usage,omitempty"`
        Services     Services           `json:"services,omitempty"`
        Backend      interface{}        `json:"backend,omitempty"`                              Servers      []ServerProperties `json:"servers,omitempty"`
}
```

Comment (harshavardhana) [reply cmd/admin-info.go]: We should keep the types @vadmeste can you fix the server first?

**Comment by vadmeste** [comment]
@harshavardhana if we are going to keep the old API, let's merge this first.. since mc admin info is wrongly showing offline nodes in the output.

**Comment by harshavardhana** [comment]
> @harshavardhana if we are going to keep the old API, let's merge this first.. since mc admin info is wrongly showing offline nodes in the output.

Yes agreed lets move forward @vadmeste

**Pull request merged**